tsql/TSqlParser.g4

```diff
@@ -3992,6 +3993,7 @@ simple_id
     | SNAPSHOT
     | SOURCE
     | SPATIAL_WINDOW_MAX_CELLS
+    | SPLIT
```

Comment:
I think introduced `STRING_AGG` also should be added here as an alternative to `simple_id`.

Reply:
Didn't think this would work, but just tried it and it does. Will add